src/options/settings/components/SearchInjection.js

```diff
@@ -14,10 +14,11 @@ const SearchInjection = ({ children }) => {
                 Want others?{' '}
                 <a
                     target="_new"
-                    href="https://github.com/WorldBrain/Memex/blob/master/CONTRIBUTING.md"
+                    href="https://github.com/WorldBrain/Memex/blob/master/GETTING-STARTED.md"
```

Comment:
should be "develop" not "master" as our main branch is 'Develop'

Reply:
@oliversauter I think it should me `master` because `master` is in the production. `develop` is just for development, we cannot give the link of development in production.

Reply:
@mukeshkharita @oliversauter So what should be the final choice?

The content of `master` and `develop` differs by a single command:

**Clone Storex submodules that we depend on**
`git submodule update --init --recursive`

Reply:
The thing is that develop is our main repo, and actually the link should therefore be also relational. Means it should show "develop" in develop and "master" in master. is that possible?

Reply:
@oliversauter I don't think so, here `master` make sense because we always push `master` branch in production.

Reply:
@mukeshkharita Note though that this is a link to GitHub. On GitHub, our main branch (i.e., the branch showed when you browse to https://github.com/WorldBrain/Memex and click on different docs, without manually changing the branch) is `develop`. Docs updates also go immediately into `develop`, before trickling into `master` when we do a release/hotfix.

It would be ideal to have a link here without specifying a branch, and just have GH redirect the user to whatever the default branch is, but I don't think that's possible. `develop` makes most sense here IMO

Reply:
Thank you @poltak for clarifying the doubt :)
The last commit that I made was for `develop` branch so I think the issue has been fixed for now
If any other changes are to be made, do let me know :D